### PR TITLE
GCM sink option to export only autoscalink metrics

### PR DESCRIPTION
**Pull request by mwielgus**
Google-influxdb k8s monitoring mode should export only autoscaling metrics.

### CODE REVIEW

File: metrics/sinks/gcm/gcm.go [hunk 1]
```diff
@@ -133,6 +151,8 @@ func (sink *gcmSink) Stop() {
 
 // Adds the specified metrics or updates them if they already exist.
 func (sink *gcmSink) register(metrics []core.Metric) error {
+
+metrics_loop:
```
Comment (piosz) [inline metrics/sinks/gcm/gcm.go]: remove this label

File: metrics/sinks/gcm/gcm.go [hunk 2]
```diff
@@ -79,10 +87,20 @@ func (sink *gcmSink) getTimeseriesPoint(timestamp time.Time, labels map[string]s
 
 	finalLabels := make(map[string]string)
 	if core.IsNodeAutoscalingMetric(metric) {
+		// All and autoscaling.
```
Comment (piosz) [inline metrics/sinks/gcm/gcm.go]: Add comment why this dead code exists here.

**Comment by piosz** [comment]
LGTM

File: metrics/sinks/gcm/gcm.go (hunk 1 above)
Comment (mwielgus) [inline metrics/sinks/gcm/gcm.go]: Done.

File: metrics/sinks/gcm/gcm.go (hunk 2 above)
Comment (mwielgus) [inline metrics/sinks/gcm/gcm.go]: Done.

**Pull request merged**